dev/make-distribution.sh

```diff
@@ -42,6 +42,8 @@ function exit_with_usage {
   echo "usage:"
   cl_options="[--name] [--tgz] [--mvn <mvn-command>]"
   echo "make-distribution.sh $cl_options <maven build options>"
+  echo "Decrease build times by adding the -T option: -T 1C will result in Maven using one thread per core."
+  echo "For example, dev/make-distribution.sh -T 1C --name hadoop-2.7.2 -Psparkr -Pyarn -Phadoop-2.7 -Phive -Phive-thriftserver -Dscala-2.11"
```

Comment:
Doesn't -T 1C have to go with the other Maven args? Also if this is just mean to show that option, maybe best to not show all these other profiles in the example. Is --name hadoop-2.7.2 valid vs just hadoop-2.7?